decoder/vaapidecoder_vp8.cpp

```diff
@@ -542,6 +544,13 @@ YamiStatus VaapiDecoderVP8::decode(VideoDecodeBuffer* buffer)
             status = ensureContext();
             if (status != YAMI_SUCCESS)
                 return status;
+            m_gotKeyFrame = true;
+        }
+        else {
+            if (!m_gotKeyFrame) {
+                //we can't decode frame without key
```

Comment:
Seems other codec may have similar issue, do we need to extend m_gotKeyFrame to other codecs?